src/portfolios/portfolios.service.ts

```diff
@@ -156,4 +157,12 @@ export class PortfoliosService {
 
     return this.transactionsService.submit(portfolio.quitCustody, {}, args);
   }
+
+  public async createdAt(did: string, portfolioId: BigNumber): Promise<EventIdentifier | null> {
+    if (portfolioId.lte(0)) {
+      throw new AppValidationError('Cannot get event details for Default Portfolio');
```

Comment:
Shouldn't we be able to return a `createdAt` date for the default Portfolio, even if its the same as the Identity?

I am guessing the SDK / middleware doesn't support  so we need this check though. OK to leave if thats the case